### ISSUE TITLE
AttributeError: type object 'EWallet' has no attribute 'create_ewallet_charge'

### ISSUE THREAD

**Issue by boazcstrike** [xendit/xendit-python#64]
For some reason, it cannot see the create_ewallet_charge function
![image](https://github.com/xendit/xendit-python/assets/8405969/04213476-73bf-4bcc-9e5b-01bdd6d650c9)

I cannot find create_ewallet_charge anywhere, here's the dir class in shell

```
>>> xendit_init = xendit.Xendit(settings.XENDIT_SECRET_KEY)
>>> dir(xendit_init)
['Balance', 'BatchDisbursement', 'CardlessCredit', 'CreditCard', 'DirectDebit', 'Disbursement', 'EWallet', 'Invoice', 'PaymentMethod', 'PaymentRequest', 'Payout', 'QRCode', 'RecurringPayment', 'Refund', 'RetailOutlet', 'VirtualAccount', 'XenPlatform', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__']
>>> dir(xendit_init.EWallet)
['Query', '__annotations__', '__class__', '__dataclass_fields__', '__dataclass_params__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__match_args__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__']
```

Python 3.10.10

**Comment by boazcstrike** [comment]
Temporary Solution - call the static method directly:
```
from xendit.models.ewallet.ewallet import EWallet
ewallet_charge_response = EWallet.create_ewallet_charge(
     reference_id="the-reference-id",
     currency="PHP",
     amount=le_amount,
     checkout_method="ONE_TIME_PAYMENT",
     api_key=XENDIT_SECRET_KEY,
     channel_code=le_payment_channel_code,
     channel_properties={
         "success_redirect_url": "http://localhost/.../",
         "failure_redirect_url": "http://localhost/.../",
     }
)
```

One hunch with @rayviloria is this is where it went wrong:
 `_XenditParamInjector`, `def instantiate`, and `_inject_function`

**Comment by adityarx** [comment]
Sorry about this @boazcstrike, I think this is an import issue with conflicting EWallet classes. I have raised a PR #65, but it might take a while to get reviewed and released since its a holiday week.

Hope your workaround is alright in the meantime 🙏

**Comment by adityarx** [comment]
@boazcstrike fix released in v0.2.2. Sorry about the delay~
